docs/example.ts

```diff
@@ -86,191 +131,329 @@ async function setup(): Promise<{
 
   // ------------------------- Claimer  ----------------------------------------
   // How to generate an Identity
-  // const mnemonic = Kilt.Identity.generateMnemonic()
-  const claimer = Kilt.Identity.buildFromMnemonic(
+  // const mnemonic = Kilt.Utils.UUID.generate()
+  const claimerMnemonic =
     'wish rather clinic rather connect culture frown like quote effort cart faculty'
-  )
-  // const address = claimer.address
+
+  // Create a light DID from the generated authentication key.
+  const claimerSigningKeypair = await keystore.generateKeypair({
+    alg: Kilt.Did.SigningAlgorithms.Ed25519,
+    seed: claimerMnemonic,
+  })
+  const claimerEncryptionKeypair = await keystore.generateKeypair({
+    alg: Kilt.Did.EncryptionAlgorithms.NaclBox,
+    seed: claimerMnemonic,
+  })
+  // Create a light DID from the generated authentication key.
```

Comment:
Same here...

Reply:
`// Using the generated authentication and encryption keys to derive a light DID.`